examples/simultaneous_translation/scripts/start-local-eval.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e
```

Comment:
Can you explain the advantages?

Reply:
In `cmd1 | cmd2` line, if `cmd1` fails, the script won't fail as the return value would be taken from `cmd2`. `set -o pipefail
` ensures the script would fail when `cmd1` fails.

For a more detailed discussion, see https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/